api/organisations/chargebee/types.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from enum import Enum
+
+
+@dataclass
+class ChargebeeObjMetadata:
```

Comment:
I think having a file called `types` is not a common pattern in python? Should we call this `metadata` instead perhaps?

Reply:
Ah, I still have a hangover from go :)

Reply:
Done